src/databricks/labs/lakebridge/deployment/job.py

```diff
@@ -157,3 +164,81 @@ def parse_package_name(self, wheel_path: str) -> str:
+        )
+        logger.info(f"Profiler ingestion job deployed with job_id={job_id}")
+        logger.info(f"Job URL: {self._ws.config.host}#job/{job_id}")
+        self._install_state.save()
+
+    def _update_or_create_profiler_ingestion_job(
+        self, name: str, source_tech: str, databricks_user: str, volume_upload_location: str, target_catalog: str
+    ) -> str:
+        job_settings = self._profiler_ingestion_job_settings(
+            name, source_tech, databricks_user, volume_upload_location, target_catalog
+        )
+        if name in self._install_state.jobs:
+            try:
+                job_id = int(self._install_state.jobs[name])
+                logger.info(f"Updating configuration for job `{name}`, job_id={job_id}")
+                self._ws.jobs.reset(job_id, JobSettings(**job_settings))
+                return str(job_id)
+            except InvalidParameterValue:
+                del self._install_state.jobs[name]
+                logger.warning(f"Job `{name}` does not exist anymore for some reason")
+                return self._update_or_create_profiler_ingestion_job(
+                    name, source_tech, databricks_user, volume_upload_location, target_catalog
+                )
+
+        logger.info(f"Creating new job configuration for job `{name}`")
+        new_job = self._ws.jobs.create(**job_settings)
+        assert new_job.job_id is not None
+        self._install_state.jobs[name] = str(new_job.job_id)
+        return str(new_job.job_id)
+
+    def _profiler_ingestion_job_settings(
+        self, job_name: str, source_tech: str, databricks_user: str, volume_upload_location: str, target_catalog: str
+    ) -> dict[str, Any]:
+        latest_lts_spark = self._ws.clusters.select_spark_version(latest=True, long_term_support=True)
+        version = self._product_info.version()
+        version = version if not self._ws.config.is_gcp else version.replace("+", "-")
+        tags = {"version": f"v{version}"}
+        if self._is_testing():
+            # Add RemoveAfter tag for test job cleanup
+            date_to_remove = self._get_test_purge_time()
+            tags.update({"RemoveAfter": date_to_remove})
+
+        return {
+            "name": self._name_with_prefix(job_name),
+            "tags": tags,
+            "job_clusters": [
+                JobCluster(
+                    job_cluster_key="Lakebridge_Profiler_Ingestion_Cluster",
+                    new_cluster=compute.ClusterSpec(
+                        data_security_mode=compute.DataSecurityMode.USER_ISOLATION,
+                        spark_conf={},
+                        node_type_id=self._get_default_node_type_id(),
+                        autoscale=compute.AutoScale(min_workers=2, max_workers=3),
+                        spark_version=latest_lts_spark,
+                    ),
+                )
+            ],
+            "tasks": [
+                NotebookTask(
```

Comment:
@sundarshankar89 's comment from PR#2000 "There are 2 ways we can implement this, have the ingestion job as python package and use a wheel task Or have the notebook upload and then run the jobs.I prefer option 1."